layouts/partials/header.html

```diff
@@ -11,12 +11,12 @@
     </div>
 </nav>
 
-<!-- <div class="bg-blue text-white text-center px-8 md:px-0 py-4 font-bold">
+<!-- <div class="bg-blue-600 text-white text-center px-8 md:px-0 py-4">
```

Comment:
Are we going to need to uncomment this before we do the cutover? I don't know how long we're planning to keep the banner up.

Reply:
I wasn't sure, either, yeah.